includes/utilities.inc

```diff
@@ -17,21 +17,38 @@ function islandora_usage_stats_log_object_view(AbstractObject $object) {
     if (($last_viewed + variable_get('islandora_usage_stats_cooldown_seconds', 300)) < REQUEST_TIME) {
       module_load_include('inc', 'islandora_basic_collection', 'includes/utilities');
       module_load_include('inc', 'islandora_usage_stats', 'includes/db');
-      $pid_id = islandora_usage_stats_get_pid_id($object->id);
-      $object_access_id = islandora_usage_stats_object_view_to_db($pid_id);
+      try {
+        $pid_id = islandora_usage_stats_get_pid_id($object->id);
+        $object_access_id = islandora_usage_stats_object_view_to_db($pid_id);
+      }
+      catch (Exception $e) {
+        watchdog('islandora_usage_stats', 'Error: islandora_usage_stats_pid_to_db() could not load %pid', array('%pid' => $object->id), WATCHDOG_ERROR);
```

Comment:
@bryjbrown almost! I'm pretty sure this won't work. If the object does not exists, then 
```PHP catch (Exception $e) {
watchdog('islandora_usage_stats', 'Error: islandora_usage_stats_pid_to_db() could not load 
%pid', array('%pid' => $object->id), WATCHDOG_ERROR); 

````
will fail because` $object->id` will never exist. 

I would suggest maybe to go to the root of the issue and never call islandora_usage_stats_log_object_view() for non existing objects?